geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/MessageDispatcher.java

```diff
@@ -533,21 +534,24 @@ private boolean handleAuthenticationExpiredException(AuthenticationExpiredExcept
       throws InterruptedException {
     long reAuthenticateWaitTime =
         getSystemProperty(RE_AUTHENTICATE_WAIT_TIME, DEFAULT_RE_AUTHENTICATE_WAIT_TIME);
+    // for old client, don't wait for re-auth but unregister this proxy completely.
+    if (getProxy().getVersion().isOlderThan(RE_AUTHENTICATION_START_VERSION)) {
+      synchronized (_stopDispatchingLock) {
+        logger.warn(OLD_CLIENT_AUTHENTICATION_EXPIRED);
```

Comment:
Why is this log message extracted as a constant? We don't seem to do this for any other log messages.

Reply:
cherry picked from Dan's PR. I guess it's not needed to be a constant.